File: sdk/javascript/examples/discovery.ts
import { Gapwise } from "@gapwise/sdk";
const gapwise = new Gapwise();
const page = await gapwise.places.list({ building: "HM", kind: "library" });
for (const place of page.data) {
  const source = place.metadataProvenance;
  console.log(
    `${place.name}: ${place.availability.state} (${source.status}, observed ${source.observedAt})`,
  );
}

File: sdk/python/examples/discovery.py
from gapwise import Gapwise

with Gapwise() as gapwise:
    page = gapwise.places.list(building="HM", kind="library")
    for place in page.items:
        provenance = place["metadataProvenance"]
        print(place["name"], place["availability"]["state"], provenance["status"])
